primitives/src/v1.rs

```diff
@@ -671,6 +671,45 @@ pub enum CandidateEvent<H = Hash> {
 	CandidateTimedOut(CandidateReceipt<H>, HeadData),
 }
 
+/// Information about validator sets of a session.
+#[derive(Clone, Encode, Decode)]
+pub struct SessionInfo {
+	/// Validators in canonical ordering.
+	#[codec(index = "0")]
```

Comment:
This doesn't make any sense for structs. It is also weird that we accept this 🤔

Reply:
Ah, I see. Good catch!